README.md

```diff
@@ -103,13 +107,23 @@ At present, Convene is split into three modules:
 - `web` which provides the human interface for Convene.
 - `api` which stores data long-term and provides a programmatic interface.
```

Comment:
Out of curiosity, for my own education, @zspencer is this `api` served by the Rails server in `convene-web`? Or is this somewhere else?

Reply:
At this point, we have not started work on the API; but it makes sense (to me) to include it in the `convene-web` project.